src/main/java/it/gov/pagopa/payhub/activities/connector/sendnotification/config/SendApisHolder.java

```diff
@@ -0,0 +1,59 @@
+package it.gov.pagopa.payhub.activities.connector.sendnotification.config;
+
+import it.gov.pagopa.payhub.activities.config.RestTemplateConfig;
+import it.gov.pagopa.pu.sendnotification.controller.ApiClient;
+import it.gov.pagopa.pu.sendnotification.controller.BaseApi;
+import it.gov.pagopa.pu.sendnotification.controller.generated.SendApi;
+import jakarta.annotation.PreDestroy;
+import org.springframework.boot.web.client.RestTemplateBuilder;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestTemplate;
+
+@Service
```

Comment:
always use @Lazy on business @Service in the repo activites
```suggestion
@Lazy
@Service
```